deployments/main.tf

```diff
@@ -31,24 +31,49 @@ module "dpe-sandbox-spacelift-development" {
   k8s_stack_deployments_name         = "DPE DEV Kubernetes Deployments"
   k8s_stack_deployments_project_root = "deployments/stacks/dpe-k8s-deployments"
 
+  auth0_stack_name         = "DPE DEV Auth0"
+  auth0_stack_project_root = "deployments/stacks/dpe-auth0"
+  auth0_domain             = "dev-sage-dpe.us.auth0.com"
+  auth0_clients = [
+    {
+      name        = "bfauble - automation"
+      description = "App for testing signoz"
+      app_type    = "non_interactive"
+    },
+    {
+      name        = "schematic - Github Actions"
+      description = "Client for Github Actions to export telemetry data"
+      app_type    = "non_interactive"
+    },
+    {
+      name        = "schematic - Dev"
+      description = "Client for schematic deployed to AWS DEV to export telemetry data"
+      app_type    = "non_interactive"
+    },
+  ]
+
```

Comment:
This is how we'll set up new account to get access to send telemetry data and authenticate with Auth0. It also relates to some of the documentation I wrote here: https://github.com/Sage-Bionetworks/schematic/pull/1527/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5